packages/contracts/hardhat.config.ts

```diff
@@ -18,9 +20,6 @@ if (!mnemonic) {
 }
 
 const infuraApiKey: string | undefined = process.env.INFURA_API_KEY;
-if (!infuraApiKey) {
-  throw new Error("Please set your INFURA_API_KEY in a .env file");
```

Comment:
Don't remove this. This check is needed for running deployments. If you require env vars, it's good practice to validate that they are there.

Reply:
okay, what if I added a variable to the hardhat.config just a bool that is "useInfuraForTests", that can be true or false?